backend/gradle/libs.versions.toml

```diff
@@ -0,0 +1,73 @@
+[plugins]
+
+com-expediagroup-graphql = { id = "com.expediagroup.graphql", version.ref = "graphql" }
+com-google-protobuf = { id = "com.google.protobuf", version = "0.9.5" }
+io-gitlab-arturbosch-detekt = { id = "io.gitlab.arturbosch.detekt", version = "1.23.8" }
+org-jetbrains-kotlin-jvm = { id = "org.jetbrains.kotlin.jvm", version = "2.1.20" }
+org-jetbrains-kotlinx-kover = { id = "org.jetbrains.kotlinx.kover", version = "0.9.1" }
+org-jlleitschuh-gradle-ktlint = { id = "org.jlleitschuh.gradle.ktlint", version = "12.2.0" }
+
+[versions]
+
+# FIXME: Keeping this at 6.10.2 since all versions from 7.0.0 onwards produce an error on tests
```

Comment:
I have added a fix for that
https://github.com/digitalfabrik/entitlementcard/pull/2146

Reply:
Thanks, I included it in this PR.

Reply:
I cherry-picked your fix commit in this PR, thanks!